core-common/src/main/java/org/glassfish/jersey/message/internal/HeaderUtils.java

```diff
@@ -47,6 +53,20 @@
 
     private static final Logger LOGGER = Logger.getLogger(HeaderUtils.class.getName());
 
+    private static final Set<String> HOP_BY_HOP_HEADERS = new HashSet<>(32);
+    private static final Pattern CONNECTION_VALUE_SPLITTER = compile("\\s*+,[\\s,]*+");
+
+    static {
+        HOP_BY_HOP_HEADERS.add("connection");
+        HOP_BY_HOP_HEADERS.add("keep-alive");
+        HOP_BY_HOP_HEADERS.add("proxy-authenticate");
+        HOP_BY_HOP_HEADERS.add("proxy-authorization");
+        HOP_BY_HOP_HEADERS.add("te");
+        HOP_BY_HOP_HEADERS.add("trailer");
```

Comment:
`Trailers` is the header name

Reply:
Have I missed something?
* [RFC2616](https://tools.ietf.org/html/rfc2616#section-14.40)
* [RFC7230](https://tools.ietf.org/html/rfc7230#section-4.4)
* [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Connection) documentation talk about the `Trailer` header.

Reply:
My bad. [RFC2616 Section 13.5.1](https://tools.ietf.org/html/rfc2616#page-92) talks about "trailers". I have not checked the [errata](https://www.rfc-editor.org/errata/eid4522).